runtime/src/comm/ugni/comm-ugni.c

```diff
@@ -6799,7 +6804,7 @@ void acquire_comm_dom(void)
 
   want_cdi = want_cdi_start = comm_dom_free_idx;
   want_cd = &comm_doms[want_cdi];
-
+  int spins = 0;
```

Comment:
This spin-wait should take even less time than the 1- to 2-microsecond network response time, because it's just waiting for a so-called comm domain to free up, and comm domains are only held long enough to initiate network transactions, not to wait for them to complete.  The latency of this function, which allocates a comm domain, has a direct impact on Chapel performance when we have more tasks than threads.  (Speaking of which, @ronawho , did you run any QSBR perf testing with such overloading?)

Reply:
Can I have a list of names for all benchmarks that I should run for multi-locale testing? All I know about is SSCA2

Reply:
Performance testing on the big Crays can be complicated, especially for marginal testing like what I mentioned above.  As long as @ronawho remains willing to run this stuff for you I'd just keep doing that, and especially as long as he's saying the results look okay.